server/src/main/scala/Routes.scala

```diff
@@ -0,0 +1,10 @@
+package server
+
+import akka.http.scaladsl.server.{Directives, Route}
+
+object Routes extends Directives {
+  def apply(): Route =
+    pathSingleSlash(getFromResource("content/index.html")) ~
+    path(Segment)(s => getFromResource(s"content/$s")) ~
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal " appears 2 times in the file.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2596262603/issues/source?bid=3301982&fileBranchId=3306761#l8)